### DISCUSSION TITLE
How do I learn how to use Gitu?

### DISCUSSION THREAD

**Discussion by superDuperCyberTechno** [altsem/gitu#200] [General]
I like the idea of Gitu a lot but I have never used Magit. So how do I learn how to do anything?

Do I simply refer to the Magit manual (this seems counterintuitive seeing as Gitu is Vim-like, where Magit is for Emacs)?

**Comment by altsem** [comment]
Valid point!

I'm considering if a "Getting started" page. Or something interactive a'la vimtutor would be intuitive.

To begin with:
Perhaps explaining how to navigate as well as the basic commands.

I'll think about it.
